lxd/daemon.go

```diff
@@ -1336,6 +1337,20 @@ func (d *Daemon) init() error {
 		}
 	}
 
+	// Setup OpenFGA authorization.
+	if openfgaAPIURL != "" && openfgaStoreID != "" && openfgaAPIToken != "" {
+		if openFGAAuthorizationModelID == "" {
+			// We should never be missing the model ID at start up if we have other connection details (this means
+			// something went wrong the last time we tried to set it up).
+			logger.Warn("OpenFGA authorization driver is misconfigured, skipping...")
```

Comment:
So this will fallback to TLS only mode or only allow unix socket operation?

Reply:
I think either both should be warnings or both should be errors, as they both result in the authorizer not working right?